dependencies.yaml

```diff
@@ -0,0 +1,149 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: conda
+    matrix:
+      cuda: ["11.8"]
+      arch: [x86_64]
+    includes:
+      - build
+      - checks
+      - cudatoolkit
+      - run
+      - test_python
+  test_python:
+    output: none
+    includes:
+      - cudatoolkit
+      - py_version
+      - test_python
+  test_cpp:
+    output: none
+    includes:
+      - cudatoolkit
+  checks:
+    output: none
+    includes:
+      - checks
+      - py_version
+  docs:
+    output: none
+    includes:
+      - cudatoolkit
+      - docs
+      - py_version
+channels:
+  - rapidsai
+  - rapidsai-nightly
+  - conda-forge
+  - nvidia
+dependencies:
+  checks:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - black=22.3.0
+          - clang=11.1.0
+          - clang-tools=11.1.0
+          - cmake-format
+          - flake8=3.8.3
+          - isort=5.6.4
+          - mypy=0.782
```

Comment:
All this can be dropped after #154. Only `pre-commit` should remain.
```suggestion
```

Reply:
fixed